packages/frontend/editor-ui/src/components/RunData.vue

```diff
@@ -268,8 +268,12 @@ const displayModes = computed(() => {
 
 	if (
 		isPaneTypeOutput.value &&
-		activeNode.value?.type === HTML_NODE_TYPE &&
-		activeNode.value.parameters.operation === 'generateHtmlTemplate'
+		(activeNode.value?.type === HTML_NODE_TYPE ||
```

Comment:
This could do with being improved, Mybe something like the below to group the checks.

```
(activeNode.value?.type === HTML_NODE_TYPE && activeNode.value?.parameters.operation === 'generateHtmlTemplate') || (activeNode.value?.type === 'n8n-nodes-base.extractFromFile' && activeNode.value?.parameters.operation === 'docx' && activeNode.value.parameters.options as Record<string, unknown>)['outputFormat'] ===
					'html')
```